tools/windows-node-installer/README.md

```diff
@@ -96,12 +104,13 @@ Flags:
 
 Global Flags:
       --credential-account string   account name of a credential used to create the OpenShift Cluster specified in 
-      the provider's credentials file (required).
+      the provider's credentials file. (required)
```

Comment:
If you put the period in front of the parenthesis, maybe you should the same for line111 and the other that doesn't have a period.

Reply:
Ideally for help output, `.` is not required. I suggest you remove it from all places.

Reply:
Ok